src/lib/update-alternatives/model/alternative.rb

```diff
@@ -58,17 +58,17 @@ def self.load(name)
       alternative = parse_to_map(raw_data.slice(0..raw_data.find_index("\n")))
       choices = raw_data.slice(raw_data.find_index("\n") + 1..raw_data.length)
       new(
-        alternative["Name:"],
-        alternative["Status:"],
-        alternative["Value:"],
+        alternative["Name"],
+        alternative["Status"],
+        alternative["Value"],
         load_choices_from(choices)
       )
     end
 
     def self.parse_to_map(alternative_data)
       alternative = {}
       alternative_data.each do |line|
-        alternative[line.split.first] = line.split.last
+        alternative[line.split(/\W/).first] = line.split.last
```

Comment:
it is a bit confusing to use different splits here. Use in both same split argument or at least write comment why different split is needed.